manifests/fips.pp

```diff
@@ -0,0 +1,58 @@
+# Manages FIPS
+#
+# @params use_fips Boolean
+#   If enabled, the system will be FIPS 140-2 enabled.
+#
+# @params use_fips_aesni Boolean
+#   If enabled and $use_fips is true, then install dracut-fips-aesni
+#
+class simp::fips (
+  Boolean $use_fips       = $::simp::use_fips,
+  Boolean $use_fips_aesni = $::cpuinfo and member($::cpuinfo['processor0']['flags'],'aes'),
+) {
+  if $use_fips {
```

Comment:
How would a user turn off fips, then? I had the same thought but the code allows to disabling it. 

That being said, the code calling it in init.pp is still wrong.